internal/compiler/generator/cpp.rs

```diff
@@ -3268,6 +3268,7 @@ fn compile_expression(expr: &llr::Expression, ctx: &EvaluationContext) -> String
                         '&' => "&&",
                         '|' => "||",
                         '/' => "/(float)",
+                        '-' => "-(float)", // conversion to float to avoid overflow between unsigned
```

Comment:
Oh, so that'll a float subtraction even if the operands are ints?

Reply:
Yes.